<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoOptionsWidgetData</class>
 <widget class="QWidget" name="HistoOptionsWidgetData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Background color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="tlp::ColorButton" name="backColorButton"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="uniformQuantificationCB">
        <property name="toolTip">
         <string>adapts the intervals on the X axis to obtain a uniform quantification on the Y axis</string>
        </property>
        <property name="text">
         <string>Uniform Quantification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cumulFreqHisto">
        <property name="toolTip">
         <string>cumulates the values obtained from one interval to the next</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Cumulative frequencies histogram       </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>controls in how many intervals the X axis repartition is distributed.</string>
          </property>
          <property name="text">
           <string>Number of histogram bins</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="nbHistoBins">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>controls in how many intervals the X axis repartition is distributed.</string>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="toolTip">
           <string>indicates the width of the small intervals on the X axis.</string>
          </property>
          <property name="text">
           <string>Resulting bin width</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="binWidth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>indicates the width of the small intervals on the X axis.</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showEdgesCB">
        <property name="toolTip">
         <string>displays the edges existing between each bin</string>
        </property>
        <property name="text">
         <string>Show graph edges above histogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>X axis</string>
        </property>
        <property name="groupTitle" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="xAxisFrame">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>indicates the number of tick marks displayed along the X axis</string>
             </property>
             <property name="text">
              <string>Tick count</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="nbXGraduations">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>indicates the number of tick marks displayed along the X axis</string>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="useXScaleCheckBox">
           <property name="toolTip">
            <string>indicates a specific range of values along the X axis</string>
           </property>
           <property name="text">
            <string>use custom range</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="useXMinLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="useXMinSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-99990000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99990000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="useXMaxLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="useXMaxSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-99990000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99990000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="xAxisLogscale">
           <property name="toolTip">
            <string>applies a logarithm scale on the X axis</string>
           </property>
           <property name="text">
            <string>use log scale</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Y axis</string>
        </property>
        <property name="groupTitle" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="yAxisFrame">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>indicates the number of tick marks displayed along the X axis</string>
             </property>
             <property name="text">
              <string>Tick step</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="YAxisIncrementStep">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>indicates the number of tick marks displayed along the X axis</string>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="useYScaleCheckBox">
           <property name="toolTip">
            <string>indicates a specific range of values along the Y axis</string>
           </property>
           <property name="text">
            <string>use custom range</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="useYMinLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="useYMinSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-99990000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99990000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="useYMaxLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="useYMaxSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-99990000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99990000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="yAxisLogscale">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>applies a logarithm scale on the Y axis</string>
           </property>
           <property name="text">
            <string>use log scale</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlp::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">tulip/ColorButton.h</header>
   <slots>
    <signal>colorChanged(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
